evm/src/utils.rs

```diff
@@ -1,4 +1,8 @@
-use ethers::prelude::{H256, U256};
+use ethers::{
+    abi::{Abi, FixedBytes, Function},
+    prelude::{H256, U256},
+};
+use eyre::ContextCompat;
```

Comment:
I believe this should just be Context

```suggestion
use eyre::Context;
```